sopel-modules/rat-board.py

```diff
@@ -938,23 +985,34 @@ def _keyfn(rescue):
             continue
         num = len(cases)
         s = 's' if num != 1 else ''
-        t = []
-        t.append("{num} {name} case{s}".format(num=num, name=name, s=s))
+        tmpOutput = []
+        tmpOutput.append("{num} {name} case{s}".format(num=num, name=name, s=s))
         if expand:
             # list all rescues and replace rescues with IGNOREME if only unassigned rescues should be shown and the
             # rescues have more than 0 assigned rats
+            # will also replace every rescue that shouldnt not be shown based on the supplied platform
```

Comment:
shouldn't not? Double negation?

Reply:
Yeah....  No, it's supposed to be a single negation.